processing/src/main/java/io/druid/segment/data/CompressedLongsIndexedSupplier.java

```diff
@@ -216,6 +216,8 @@ public void remove()
 
   private class CompressedIndexedLongs implements IndexedLongs
   {
+    final Indexed<ResourceHolder<LongBuffer>> cachedLongBuffers = baseLongBuffers.singleThreaded();
```

Comment:
This seems like a really weird name for this object given that it won't actually be cached by the object.  That's made even weirder by the fact that `CompressedIndexedLongs` is actually caching the result already.